client/image_load.go

```diff
@@ -2,32 +2,74 @@ package client // import "github.com/docker/docker/client"
 
 import (
 	"context"
+	"fmt"
 	"io"
 	"net/http"
 	"net/url"
 
 	"github.com/docker/docker/api/types/image"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+// ImageLoadOption is a type representing functional options for the image load operation.
+type ImageLoadOption interface {
+	Apply(*imageLoadOpts) error
+}
```

Comment:
Small nit; not sure what we did for the other ones, but perhaps we should choose to put the options in a separate file (also if we at some point would decide to split "clients" to smaller ones as we were contemplating); e.g. `image_load_options.go`

That way, if things would be split, we can make these a lateral move to a sub-package (and an alias in return)

Reply:
☝️ discussion with @vvoland to see what makes most sense; also because the client already has "one file per function", so it's not as bad as some other cases we had in the past where tons of code and options where sprinkled together